### DISCUSSION TITLE
封面如何引用icarus路径外的图片

### DISCUSSION THREAD

**Discussion by Fire-Knight** [ppoffice/hexo-theme-icarus#1169] [Q&A] [answered]
我想给文章做封面，但好像只能使用主题目录下的图片，可否使用md文件同层的图片？
例如下图所示，在A.md中，使用cover: /img/A2.png，封面是可以正常显示的。但无法访问到项目的source下的/_post/A1.png。
一般而言，图片和笔记是在一块的。请问是我的项目组织有问题，还是cover路径有误？

```
| - scaffods
| - source
    | - _post
        | - A1.png
        | - A.md
| - themes
    | - icarus
        | - source
            | - img
                | - A2.png
```

**Comment by ppoffice** [comment]
@Fire-Knight  https://hexo.io/docs/asset-folders

**Reply by Fire-Knight** [reply]
根据文档以及一些博客，我又进一步尝试，发现问题在于：cover标签似乎无法使用_post下的图片。
如下所示，`cover: /about/A2.jpg` 、`cover: A3.jpg`均生效，但`cover: /_posts/A1.jpg` 不可行。

```
| - scaffods
| - source
    | - _posts
        | - A1.jpg
        | - A.md
    | - about
        | - A2.jpg
    | - A3.jpg
```

**Comment by Fire-Knight** [comment, marked as answer]
根据public目录下生成的内容来看，_post目录会被拆分为其他层级结构，其他目录会保留。所以可以建个文件夹来存图片什么的，同时使用基于source的绝对路径。
